### PR TITLE
Added Hebrew language to Nutshell.language

### PR DESCRIPTION
**Pull request by Adiveler**
Added Hebrew to Nutshell.language in the nutshell.js file.

### CODE REVIEW

File: nutshell.js [hunk 1]
```diff
@@ -434,6 +434,33 @@ Bubble: the box that expands below an expandable, containing a Nutshell Section
             // What punctuation (in this language) signifies the END of a sentence? Note, this is a regex.
             endPunctuation: /[。.?!]\s/g // added chinese period
 
+        },
+        he: {
+            // Button text
+            closeAllNutshells: `סגור את כל האגוזים`,
+            learnMore: `עוד אודות קליפת האגוז`,
```
Comment (Adiveler) [inline nutshell.js:441]: Might be better to keep the original English name (nutshell), rather than translate it (קליפת האגוז).

**Comment by TomLisankie** [comment]
Just so you know, it says in the README that she is not accepting pull requests.

**Comment by Adiveler** [comment]
> “That said, I'm accepting two kinds of Pull Requests! 1) Bug fixes, and 2) Translations.”

**Comment by TomLisankie** [comment]
> > “That said, I'm accepting two kinds of Pull Requests! 1) Bug fixes, and 2) Translations.”

Ah! My bad. Sorry.

**Comment by ncase** [comment]
Hi @Adiveler ! Sincerest apologies it took me two months to finally getting around to pulling this (my inbox and schedule has been a mess), I super appreciate you translating this and making Nutshell accessible to more folks. Thank you again so much! 👍

**Comment by Adiveler** [comment]
Better late than never.
Thank you very much 😃

**Pull request merged**